client/.sass-lint.yml

```diff
@@ -0,0 +1,56 @@
+####################
```

Comment:
This shouldn't be here anymore, precisely because it requires discussion with you and @gabemart - it's a leftover from me trying out sass linting. I'll remove it.